joomla/joomla-cms
# [4.0] Migrate data for repeatable fields

Pull Request for Issue #30556.

### Summary of Changes
This PR solves the Release Blocker issue #30556. It convert data of repeatable fields to the format accepted by subfield.


### Testing Instructions
1. Install Joomla 3.10.0. You can download the nightly build package here https://developer.joomla.org/nightlies/Joomla_3.10.0-alpha6-dev-Development-Full_Package.zip to install it.
2. Create one or two repeatable field for articles.
3. Create one or two articles, enter data for the created custom fields.
4. Upgrade your site to latest Joomla 4.0-dev package (generate for this PR). Just download the updated package here https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/32611/downloads/41496/Joomla_4.0.0-beta8-dev+pr.32611-Development-Update_Package.zip , then go to **Components** -> **Joomla ! Update**, look **Upload & Update** tab and install that update package.
5. Check the articles on upgraded site, edit these article, check and make sure data in Fields tab is there.


### Actual result BEFORE applying this Pull Request
Data for repeatable fields not being migrated


### Expected result AFTER applying this Pull Request
Data for repeatable fields are migrated properly
